src/test/perl/NCM/Component/brokennew.pm

```diff
@@ -0,0 +1,12 @@
+package NCM::Component::brokennew;
+
+use strict;
+use warnings;
+use LC::Exception;
+
+# Needs to hav a new method, ideally via NCM::Component inheritance
```

Comment:
OK, I understand the test now. Can this line say "All components need to have a new() method, ideally via NCM::Component inheritance." Otherwise it looks a bit like you've left a "FIXME" comment. :-)